procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+# How to set up automation for nightly Android product
+
+These instructions define which resources are needed for setting up nightly Android product release automation, as well as the steps to prepare the automation itself. Additionally, this guide include steps for running a "staging" version of the automation so that it can be tested in the developer's own repository.
+
+Throughout this document, wherever the term `$product` is used, substitute your product's name in (replacing spaces with hyphens), e.g.: `reference-browser` or `fenix`.
+
+Note: we don't need to explicitly define scopes in Taskcluster. We just define them in the task, hook's role, and each scriptworker script. Taskcluster will implicitly detect scopes, but it doesn't use them for anything other than generic equality/wildcard verification.
```

Comment:
Ah, ok, this makes sense, sorry about the misunderstanding. I still think it's worth adding a small sentence talking about the manuevring of the scopes and the control enforcing, not _defining_ them. But we can always iterate later, this is certainly not blocking.

Reply:
I don't quite understand: what kind of thing would you like me to emphasize? I'm not sure what to place about "manuevering the scopes and control enforcing" - is this about ci-admin? :thinking:

Reply:
Yes. I see this section slightly expanded:

a) existing paragraph that describes what scopes actually are and how they don't need to be defined specifically, but are strings in Taskcluster that we define in various places to ensure a good security model.

b) a follow-up paragraph like the one I mentioned above, where we complete moreover the fact that these scope administration in hooks/roles and such is done or will be done soon via tools like `ciadmin`.

Reply:
I'll add a note, though there are other changes that will affect these docs as well, such as the `pushapk` changes - rather than having a note of what _will_ change, I'd rather update the docs _when_ the change happens. This way, the docs stay tight and relevant

Reply:
That's a fair point. I guess I'm being biased since I hope to land those changes this week :)